Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -62,10 +65,12 @@ TEST(Serializer, CanLinkToNanopb) {
   pb_ostream_from_buffer(nullptr, 0);
 }
 
+const DatabaseId kDatbaseId{"p", "d"};
```

Comment:
`DatabaseId` is not trivially destructible, so this instance is against the style guide:

https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Reply:
Done.